### PR TITLE
Create understand_data.R

### PR DESCRIPTION
**Pull request by kouamea**
Aymone_snippet

### CODE REVIEW

**Review by kelseyMayo — APPROVED** [review]
Hi Aymone - this looks great. Approved!

**Review by kelseyMayo — APPROVED** [review]

File: understand_data.R [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Glimpse of data and missing value check
+understand_data<- function(data) {
```
Comment (deflaux) [inline understand_data.R:2]: nit: spaces on both sides of `<-`

You can automate finding/fixing the nit-picky stuff using the R packages described here: https://style.tidyverse.org/

File: understand_data.R [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    glimpse(data)
+    
+    check_missing = levels(factor(is.na(data)))
+    if(length(check_missing) <= 1 && check_missing != TRUE) {
+        cat(blue("There are no missing values in your dataset."))
+        } else {
```
Comment (deflaux) [inline understand_data.R:8]: nit: here and elsewhere use consistent indentation

You can automate finding/fixing the nit-picky stuff using the R packages described here: https://style.tidyverse.org/

File: understand_data.R [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Glimpse of data and missing value check
```
Comment (deflaux) [inline understand_data.R:1]: Consider using [Roxygen](http://r-pkgs.had.co.nz/man.html#man-workflow) for documentation for functions.

Then users can type `?understand_data` in their notebooks and get a really pretty view of the documentation for the function, just like functions in other R packages. (try `?na.omit`).

**Review by deflaux — COMMENTED** [review]
I tested this by running it on the dataframe created by of one of the measurements SQL snippets, looks good!

```
understand_data(most_recent_measurement_of_interest_df)
```

File: understand_data.R (hunk 3 above)
Comment (deflaux) [reply understand_data.R:1]: Correction: `?some_R_function` works for functions that are part of an R package, so it won't work in this case and that's okay, but it's still great to add the documentation for the function in Roxygen format.

**Comment by deflaux** [comment]
closing old pull requests

**Pull request closed without merging**